v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -675,3 +679,49 @@ export const curveBasis = (points: Point[]) => {
   path += pathBasis(p2, p3, p3, p3)
   return path
 }
+
+const axisPlaceToOrientationMap: Record<GraphPlace, AxisOrientation> = {
+  "bottom": "horizontal",
+  "top": "horizontal",
+  "left": "vertical",
+  "rightCat": "vertical",
+  "rightNumeric": "vertical",
+  "yPlus": "vertical",
+  "plot": "horizontal",
+  "legend": "horizontal"
+}
+
+export interface IUpdateGraphValues {
+  attributeId: string | number
+  attributeName?: string
+  axisOrientation?: string
+  plotType?: string,
+  primaryRole?: string
+}
+
+export const attrChangeNotificationValues = (
+  place: GraphPlace, attrId: string, attrName?: string, attrIdToRemove = "",
+  tile?: ITileModel
+) => {
+
+  const axisOrientation = axisPlaceToOrientationMap[place] || "axis"
```

Comment:
I'm surprised to see a default here that doesn't appear in `axisPlaceToOrientationMap`.